Parser.fs

```diff
@@ -125,6 +125,15 @@ let parseParamList argp =
  * Expressions.
  *)
 
+/// Takes a Parser<'a, 'u> and gives back an annotated AST Node parser
+/// Parser<Node<'a>, 'u> which will annotate with extra information from
+/// the stream.
+let nodify v = 
```

Comment:
`onNodes` or something?  Not a big deal really, just `nodify` is a little ambiguous imo